recipes/molecule-inspec/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "molecule-inspec" %}
+{% set version = "1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/molecule-inspec-{{ version }}.tar.gz
+  sha256: dd171044547ecb777e4c35c1fe02fbb294a9a2972a7c9b5ab8522fa24a9092a8
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or py < 37]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

It's fine if the Windows CI fails due to missing dependencies.